hi-ml-azure/src/health_azure/utils.py

```diff
@@ -55,7 +55,7 @@
 # name, hence version will always be fixed
 ENVIRONMENT_VERSION = "1"
 
-# Environment variables used for authentication
+# Environment variables used for authentication and workspace selection
```

Comment:
Can we maybe break this comment? Specify which env variables are used for auth (59-61) and add a comment line 62 for Workspace selection env vars?